Products/ZenModel/actions.py

```diff
@@ -394,6 +396,37 @@ def _encodeBody(self, body):
             plain_body = MIMEText(body.decode('ascii', 'ignore'))
         return plain_body
 
+    def _evtTimeToUserTz(self, body, zone_name):
+        """
+        Convert event time from email body to timezone from use property.
+        """
+        evt_date = ''
+        evt_date_with_tz = ''
+        try:
+            evt_date = re.search(
+                '(\d+/\d+/\d+ \d+:\d+:\d+.\d+)', body).group()
```

Comment:
Does the millisecond (or nanosecond) portion of the timestamp _always_ present?  If it doesn't, this won't match.